test/infrastructure/docker/internal/docker/manager.go

```diff
@@ -46,21 +47,21 @@ type Manager struct{}
 
 type nodeCreateOpts struct {
 	Name         string
-	Image        string
 	ClusterName  string
 	Role         string
 	EntryPoint   []string
 	Mounts       []v1alpha4.Mount
 	PortMappings []v1alpha4.PortMapping
 	Labels       map[string]string
 	IPFamily     clusterv1.ClusterIPFamily
+	kindMapping  kind.Mapping
```

Comment:
```suggestion
	KindMapping  kind.Mapping
```
Should be exported as well like the other fields